### PR TITLE
feat(onboarding): Add new footer tests - [TET-726]

### PR DESCRIPTION
**Pull request by priscilawebdev**
Add unit tests for the new wizard footer style changes

### CODE REVIEW

File: static/app/views/onboarding/components/footer.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+    renderGlobalModal();
+
+    userEvent.click(screen.getByRole('button', {name: 'Explore Sentry'}));
+
+    // Display are you sure modal
+    expect(await screen.findByText('Are you sure?')).toBeInTheDocument();
+  });
+
+  it('error processed ui', function () {
```
Comment (andriisoldatenko) [inline static/app/views/onboarding/components/footer.spec.tsx:98]: do you want to name this step differently?

Comment (priscilawebdev) [reply static/app/views/onboarding/components/footer.spec.tsx:98]: we are calling it "Error Processed!" 

image


What exactly do you mean?

Comment (andriisoldatenko) [reply static/app/views/onboarding/components/footer.spec.tsx:98]: you misunderstand my comment:
on line 57 you have the same name, so I was wondering if we can name it differently.

Comment (priscilawebdev) [reply static/app/views/onboarding/components/footer.spec.tsx:98]: I see... but on line 57 I call it "processing error ui"... slightly different :)

Comment (andriisoldatenko) [reply static/app/views/onboarding/components/footer.spec.tsx:98]: nvm i misread it.

**Review by andriisoldatenko — APPROVED** [review]
LGTM

image

**Pull request merged**